frontend/src/App.jsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import Loader from "./Components/Loader/Loader";
+
+function App() {
```

Comment:
I think that we don't need to define functions with a 'function' keyword.
Also return statement is not needed, you can just write something like:
const App = () => <Loader />